src/components/Fields/Extensions.tsx

```diff
@@ -1,4 +1,9 @@
 import * as React from 'react';
+
+import { startCase } from 'lodash';
```

Comment:
Is there a point to pull in lodash because of the only `startCase` function?

Could you implement a simple one yourself e.g. in `utils` or just copy lodash's one

Reply:
Why are you afraid of importing single function ?
I don't think it is good idea to re-invent (or copy) common functions already defined in libralies.

Reply:
It's not about importing a single function. It's about adding a new dependency as `lodash` is not in the dependencies yet.

The main reasoning behind this is the bundle size. While it doesn't matter so much for node apps it matters for browsers. Tree-shaking isn't perfect. I just tested and importing this single function leads to *68KB!* increase in bundle size. Don't you find it too much for such a simple function (especially since vendor extension names can't contain any special Unicode symbols).

Hope this does make sense.

Reply:
It makes sense 👍 
Looks like `lodash` are not used in production code right now, so importing single function may add `68KB' or around to bundle.
![image](https://user-images.githubusercontent.com/127635/54418931-9cc74300-4749-11e9-9546-cc7bc1d63b60.png)

Perhaps `lodash` dependencies should be replaced with more granular modules like [lodash.set](https://www.npmjs.com/package/lodash.set).

Reply:
Yes, but this is development code so it doesn't matter so much and I don't think it's worth to do.

Reply:
Hi @RomanGotsiy and @exoego, thanks for having a look at this.

So I agree with @exoego to avoid reinventing the wheel etc where possible, but 68k bundle increase is too big for a quite simple function. I'm ultimately happy with switching this out to something more appropriate. Happy with what you guys decide really.

Reply:
I'm not sure now if we need to title-case the extension name at all.

Vendor extension can contain arbitrary name so e.g.: `x-Some Title` is a valid extension. So you have a workaround to show title-cased labels but if we title case them by default other users don't have a way to display lower-case label.

Summing up, I think we should remove `startCase` at all. Do you agree?

Reply:
Fair point, if `x-Some Title` is valid then yeah completely agree - we can't guess or impose one method of presentation over another.

And I can solve my use case by just capitalising the first letter after the `x-` prefix, so I'm happy.

I'll update the PR in a moment